spec/features/admin/bulk_order_management_spec.rb

```diff
@@ -21,12 +21,16 @@
     end
 
     context "displaying the list of line items" do
-      let!(:o1) { create(:order_with_distributor, state: 'complete', completed_at: Time.zone.now ) }
-      let!(:o2) { create(:order_with_distributor, state: 'complete', completed_at: Time.zone.now ) }
+      let!(:o1) { create(:order_with_distributor, state: 'complete', shipment_state: 'ready', shipment_state: 'ready', completed_at: Time.zone.now ) }
+      let!(:o2) { create(:order_with_distributor, state: 'complete', shipment_state: 'ready', shipment_state: 'ready', completed_at: Time.zone.now ) }
```

Comment:
I think we have a double here: ` shipment_state: 'ready',` is appearing twice in the lines above.

Reply:
Yeah, I probably appeared when I merged. I just fixed it :smile: